.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/anthony/miniconda3/envs/q2-dev/bin/python"
```

Comment:
I'm good with us adding vscode settings files for [things like tasks](https://code.visualstudio.com/Docs/editor/tasks), but maybe its better to leave out the python env settings.